requirements.txt

```diff
@@ -1,5 +1,5 @@
 django>=1.9,<1.11
-djangorestframework>=3.3,<3.6
+djangorestframework>=3.3,<=3.6.2
```

Comment:
@rohithasrk follow the convention, that is, to put less than the next major version, that is, `<3.7`, this way we won't have to change this number at each minor release of django-rest-framework. We assume minor version do not bring backward incompatible changes.